lib/ex_vatcheck/vies_client.ex

```diff
@@ -27,7 +27,7 @@ defmodule ExVatcheck.VIESClient do
   def check_vat(client, country_code, vat_number) do
     req_body = vat_request(country_code, vat_number)
 
-    case HTTPoison.post(client.url, req_body) do
+    case HTTPoison.post(client.url, req_body, [{"content-type", "text/xml"}]) do
```

Comment:
This header addition is what fixed the 500 error described in https://github.com/taxjar/ex_vatcheck/issues/12. I'm guessing in addition to changing the XML namespacing, they also now enforce a `content-type` header.